test/node/NodeTests.md

```diff
@@ -0,0 +1,6 @@
+This folder contains unit tests which are node-specific.
+
+Karma is used for testing the code in browsers. To run the unit tests using karma run the following steps -
+
+-   Run the script npm install
```

Comment:
nit 

Run the script `npm install`
Run the script `npm run test`

Reply:
Done